Vizzuality/heco-invest
# Fix issues related to the static content translation pipeline

This PR fixes two issues related to the static content translation pipeline:
1. The application may crash when initialising due to missing compiled translation files
2. Unnecessary errors would pollute the console during development

There was a third issue that I haven't unfortunately been able to solve: if the user would use a non default locale (such as Portuguese), then they'd go to `/about` instead of `/pt/about`, not only they wouldn't be redirected to `/pt/about`, but their locale would be changed back to the default one (Spanish).

This is actually [how Next.js implemented the internationalised routing](https://nextjs.org/docs/advanced-features/i18n-routing#leveraging-the-next_locale-cookie). They may provide [changes in the future](https://github.com/vercel/next.js/issues/20657#issuecomment-753253778) to cover our needs.

## Testing instructions

1. Start the application in dev mode

The application shouldn't crash.

3. Change your locale to Portuguese
4. Open the console

You shouldn't see any error due to missing translations.

5. Build and start the application in production mode
6. Change your locale to Portuguese

You should see error due to missing translations.

## Tracking

Not tracked, but directly related to #12.
